spec/feature/show_user_view_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'rails_helper'
+RSpec.describe 'User Show Page', type: :system do
+  describe 'User show page', js: true do
+    it 'displays the the profile picture for each user.' do
+      visit('users/1')
+      page.has_content?('Screenshot Image')
+      expect(page).to have_css('img')
+    end
+    it 'displays the user profile picture.' do
+      visit('users/1')
+      expect(page).to have_content('Lilly')
+    end
+    it 'display the number of posts each user has written.' do
+      visit('users/1')
+      expect(page).to have_content('3')
+    end
+    it 'display the user bio' do
+      visit('users/1')
+      expect(page).to have_content('web developer')
+    end
+    it 'display the user first 3 posts' do
+      visit('users/1')
+      expect(page).to have_content('What are 3 interesting facts about Mexico?')
+      expect(page).to have_content('Algorithm')
+      expect(page).to have_content('What is the role of women in technology?')
+    end
+    it 'display a button that lets me view all of a user posts.' do
+      visit('users/1')
+      expect(page).to have_content('See all posts')
+    end
+    it 'When I click a users post, it redirects me to that post' do
+      visit('users/1')
+      click_button('See all posts')
+      expect(page).to have_current_path('/users/1/posts')
+    end
+    it 'it When I click a users post, it redirects me to that post' do
+      visit('users/1')
+      click_button('See all posts')
+      expect(page).to have_current_path('/users/1/posts')
+    end
+  end
+end
```

Comment:
- [ ] Just as you were told by the previous reviewer, When I run the unit tests they fail. The error being shown is ```An error occurred while loading ./spec/feature/show_user_view_spec.rb.```

![image](https://user-images.githubusercontent.com/86318284/208915892-79faeccb-b9d3-4d79-994a-12b4afd23b33.png)

Reply:
Hello @Nemwel-Boniface, Thanks for your review but all tests are passing in my side 

Here is the prove of that
![image](https://user-images.githubusercontent.com/98043302/208918400-f902e8e1-4bcc-4641-8cc7-89ceaa222071.png)
![image](https://user-images.githubusercontent.com/98043302/208919142-395974ea-8073-437f-95d7-a89c1448c718.png)